content/blog/learn-what-you-can-do-with-hpe-data-services-cloud-console-api-in-just-3-minutes.md

```diff
@@ -0,0 +1,201 @@
+
+> **Note:** This blog post assumes you have created an [HPE GreenLake account](https://console.greenlake.hpe.com/) and joined your account to your company account (also called an ***organization***). You also got assigned appropriate roles and permissions by the administrator for your organization in order to access HPE data services resources (for example storage arrays and volumes) through the Data Services Cloud Console application instances. A Data Service Cloud Console application instance is a service cluster running in one of the HPE regions.
+
+### Data Services Cloud Console and REST API
+
+Data Services Cloud Console supports a set of REST APIs that allows users to integrate HPE Data Services Cloud Console with their custom applications. By using [OAuth 2.0 protocol](https://oauth.net/2/) to authenticate and authorize applications, secure and time-limited (***120 minutes***) access to HPE data services are provided via an **access token**. The token ensures that client API requests access HPE data services for the requested operation, securely and according to the authorization granted to the user who created it.
+
+> **Note:** You can find the Data Services Cloud Console API Documentation [here](https://console-us1.data.cloud.hpe.com/doc/api/v1/) and in the help section of the [HPE GreenLake Cloud Platform](https://console.greenlake.hpe.com/).    
+
+The REST APIs support standard HTTP request methods (GET, POST, PATCH, PUT and DELETE). A HTTP request is made by providing a specific HPE regional connectivity endpoint for the Data Service Cloud Service application instance, HTTP request method, access token and data payload. The HTTP response for these requests are returned in the JSON format.
+
+Currently, there are three HPE regional Data Services Cloud Console application instance endpoints: 
+
+* EU Central    
+
+  * https://eu1.data.cloud.hpe.com   
+ 
+* AP Northeast    
+
+  * https://jp1.data.cloud.hpe.com    
+
+* US West    
+
+  * https://us1.data.cloud.hpe.com    
+
+
+HPE GreenLake Cloud Platform allows developers to make API calls on a particular regional Data Services Cloud Console customer instances. Using the API functionality in the HPE GreenLake Cloud Platform graphical user interface (GUI), developers can create their **API client application** credentials. The credentials consist of a *ClientID-ClientSecret* pair that represents the permissions granted to the user who creates the API client application credentials, to access the protected resources. The credentials is then used to generate and refresh expired OAuth based access token. Once the token is generated or refreshed, it can be used as **authorization bearer token** to make further secure REST API calls to HPE data services resources via the regional Data Services Cloud Console application instance.
```

Comment:
In the first sentence of this paragraph "instances" should be singular - "instance" as you state they can make calls on a specific instance.

3rd sentence: grammatically, you don't really need the comma prior to "to access the protected resources". However, not having the comma there, it reads a little choppy. May I suggest rewording it this way?

"The credentials consist.... who creates the API client application credentials and is used to access the protected resources."

4th sentence: because "credentials" is plural, you would use the verb "are" and so it would then be "The credentials are then used..."

5th sentence: currently missing the word "an" prior to "authorization bearer token".